filesystem/src/main/java/org/frankframework/filesystem/FileSystemUtils.java

```diff
@@ -293,20 +293,19 @@ public static <F> Stream<F> getFilteredStream(IBasicFileSystem<F> fileSystem, St
 	}
 
 
-	public static <F, FS extends IBasicFileSystem<F>> String getFileInfo(FS fileSystem, F f, DocumentFormat format) throws FileSystemException {
+	public static <F, FS extends IBasicFileSystem<F>> String getFileInfo(FS fileSystem, F f, TypeFilter typeFilter, DocumentFormat format) throws FileSystemException {
```

Comment:
Ik denk dat het niet nodig is om hier het `typeFilter` toe te voegen.
Ik denk dat de zorg om performance wat overtrokken is, zeker omdat we bij S3 geen query naar AWS hoeven te doen nu.

Altijd het `type` attribuut toevoegen maakt de output consistenter en dus makkelijker te parsen ongeacht de opties die gebruikt waren om eraan te komen; deze output wordt in best wat plaatsen gebruikt en niet alleen bij `LIST`.

Reply:
Ja, ik twijfelde ook bij deze methode eigenlijk en dacht er aan om het consistent te houden. Als dit niet nodig is, ook in de toekomst niet, kan ik er prima altijd FILES van maken. 

Qua performance is het inderdaad prima, denk ik. Voortschrijdend inzicht. Ik zal dit morgen effe aanpassen. Dank

Reply:
Wat bedoel je met "er altijd FILES van maken"?

Reply:
Aangepast.